features/project/project_controller.rb

```diff
@@ -98,6 +99,9 @@ class ProjectController < AuthenticatedControllerBase
           fastfile_path: fastfile_path
       }
 
+      # Delete the project
+      FileUtils.rm_rf(repo_path)
```

Comment:
I'm scared of this. What is `repo_path` is nil, or `` or accidentally `/` ?

Reply:
Is safe to keep the cloned repo where it is? Or how can I check if this is safe to delete?

Reply:
I think now is better 😄